css/css-multicol/column-fill-balance-orthog-block-001.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Writing Modes Test: 'column-fill: balance' of a vertical writing mode block in orthogonal context</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css3-multicol/#cf">
+  <link rel="help" href="https://www.w3.org/TR/css-writing-modes-4/#auto-multicol">
+  <link rel="match" href="reference/column-fill-balance-orthog-block-001-ref.html">
+
+  <!--
+
+  Issue 719014: column-fill: balance of a vertical block in orthogonal context incorrectly rendered
+  https://bugs.chromium.org/p/chromium/issues/detail?id=719014
```

Comment:
Could also be a <link rel="help"...

Reply:
When I was creating HTML 4 tests, I was using 
`<link rel="bookmark"...`
for linking to bug reports. HTML 5 does not support rel="bookmark".

[Writing tests guidelines § Specification Links](http://web-platform-tests.org/writing-tests/css-metadata.html#specification-links) does not mention using [<link rel="help"...](url) for linking to bug report.

Reply:
I don't know. Me and my colleagues have been linking to bugs in that manner for some time already. But up to you.